scripts/readme-gen/templates/README.tmpl.rst

```diff
@@ -0,0 +1,75 @@
+{# The following line is a lie. BUT! Once jinja2 is done with it, it will
+    become truth! #}
```

Comment:
I think I recently updated this file with some new fields.

Reply:
Updated